mindocr/models/necks/abinet_neck.py

```diff
@@ -0,0 +1,145 @@
+import numpy as np
+
+import mindspore as ms
+from mindspore import nn
+
+from ...models.utils.abinet_utils import Model, PositionalEncoding
+from ...models.utils.abinet_utils import TransformerDecoder as ms_TransformerDecoder
+from ...models.utils.abinet_utils import _default_tfmer_cfg
+
+__all__ = ["ABINet_Encoder"]
+
+
+class ABINet_Encoder(nn.Cell):
+    def __init__(self, in_channels=512):
+        super().__init__()
+        self.iter_size = 3
+        self.max_length = 26  # additional stop token
+        self.out_channels = 512
+        self.in_channels = in_channels
```

Comment:
same

Reply:
Modified, thank you.

Reply:
Mindocr requires class neck to contain out_channels. Although this model is not used, it must be retained. The rest have been modified, thank you.